src/keysignature.ts

```diff
@@ -102,16 +115,37 @@ export class KeySignature extends StaveModifier {
     };
   }
 
+  // Add this key signature to the start of a stave.
   addToStave(stave: Stave): this {
     this.paddingForced = true;
     stave.addModifier(this);
 
     return this;
   }
 
+  // Get the `BoundingBox`
+  getBoundingBox(): BoundingBox {
+    if (!this.formatted) this.format();
+
+    return super.getBoundingBox();
+  }
+
+  // Calculate the width and height for the entire signature
+  protected calculateDimensions(): void {
+    let boundingBox: BoundingBox;
+    if (this.children.length > 0) boundingBox = this.children[0].getBoundingBox();
+    else boundingBox = new BoundingBox(this.x + this.xShift, this.y + this.yShift, 0, 0);
```

Comment:
Same here.

I know it looks nice and compact, but I'm not a fan of this style.

Using braces on if/else blocks allows us to add new lines of code in the future without accidentally causing bugs.

I also personally think it is easier to read with braces in general.

Reply:
done